SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/IStorage.cs

```diff
@@ -58,6 +58,13 @@ public interface IStorage : IDisposable
         /// <exception cref="StorageException"/>
         Task<string> GetLogsAsync(string channelName, int limit, List<Log> logs);
 
+        /// <summary>
+        /// Waits for any running storage operations to complete.
+        /// </summary>
+        /// <param name="timeout">The maximum amount of time to wait for remaining tasks</param>
+        /// <returns>True if remaining tasks completed in time; false otherwise</returns>
+        Task<bool> WaitAsync(TimeSpan timeout);
```

Comment:
Either add Operations to method name or remove Operations to the Channel method to make both consistent.

Reply:
Is boolean needed? I don't think so.